lib/utils/pad-left.ts

```diff
@@ -0,0 +1,3 @@
+export function padLeft(dateValue: number) {
```

Comment:
Aww, why not https://www.npmjs.com/package/pad-left? 😭

Reply:
Good point :D I added the pad-left package